lock.go

```diff
@@ -116,16 +117,12 @@ func (l *Lock) fillUserInfo() error {
 	}
 	l.Username = usr.Username
 
-	uid, err := strconv.ParseInt(usr.Uid, 10, 32)
-	if err != nil {
-		return err
-	}
+	// We ignore the error. On Windows Uid is not a number
+	uid, _ := strconv.ParseInt(usr.Uid, 10, 32)
```

Comment:
This should also have its own function that just does nothing on Windows, instead of ignoring the error for all platfoms. Maybe something like `fillTimes()` from `node.go`, e.g. for Linux: https://github.com/restic/restic/blob/master/node_linux.go#L21